getting-started/templates/systemlink-values.yaml

```diff
@@ -512,6 +512,22 @@ dataframeservice:
   ##
   requestBodySizeLimitMegabytes: 250
 
+  ## Configure rate limiting. Limits are enforced per-replica. Each replica of the dataframe service
```

Comment:
Should this have an ATTENTION tag. Do we expect a typical user to need to consider these settings?

Reply:
I think it is very likely that a user will run into 429s if many concurrent users try to ingest data. I'll add the tag.